tests/src/Kernel/Update405Test.php

```diff
@@ -19,18 +19,25 @@ class Update405Test extends KernelTestBase {
 
   /**
    * @covers ::enableAutosaveForm
+   * @covers ::enableRedirect
    */
-  public function testEnableAutosaveForm() {
-    /** @var \Drupal\Core\Extension\ModuleHandlerInterface $module_handler */
-    $module_handler = $this->container->get('module_handler');
-    $this->assertFalse($module_handler->moduleExists('autosave_form'));
-    $this->assertFalse($module_handler->moduleExists('conflict'));
+  public function testUpdate() {
```

Comment:
I think that there might be an extra blank line here that shouldn't be.

Reply:
I'll get those coding standards things figured out asap. :)